### DISCUSSION TITLE
Error with Sacado DataType

### DISCUSSION THREAD

**Discussion by ppandit95** [peridigm/peridigm#252] [Q&A]
Dear Peridigm Community

Although I do realise that this error message pertains to trilinos library but any suggestion on the error that I am facing wherein I am getting an error as -
`error: cannot convert ‘Sacado::mpl::enable_if_c<true, Sacado::Fad::Exp::AdditionOp<Sacado::Fad::Exp::SubtractionOp<double, Sacado::Fad::Exp::MultiplicationOp<double, Sacado::Fad::Exp::GeneralFad<Sacado::Fad::Exp::DynamicStorage<double, double> >, true, false, Sacado::Fad::Exp::ExprSpecDefault>, true, false, Sacado::Fad::Exp::ExprSpecDefault>, Sacado::Fad::Exp::MultiplicationOp<double, Sacado::Fad::Exp::GeneralFad<Sacado::Fad::Exp::DynamicStorage<double, double> >, true, false, Sacado::Fad::Exp::ExprSpecDefault>, false, false, Sacado::Fad::Exp::ExprSpecDefault> >::type’ {aka ‘Sacado::Fad::Exp::AdditionOp<Sacado::Fad::Exp::SubtractionOp<double, Sacado::Fad::Exp::MultiplicationOp<double, Sacado::Fad::Exp::GeneralFad<Sacado::Fad::Exp::DynamicStorage<double, double> >, true, false, Sacado::Fad::Exp::ExprSpecDefault>, true, false, Sacado::Fad::Exp::ExprSpecDefault>, Sacado::Fad::Exp::MultiplicationOp<double, Sacado::Fad::Exp::GeneralFad<Sacado::Fad::Exp::DynamicStorage<double, double> >, true, false, Sacado::Fad::Exp::ExprSpecDefault>, false, false, Sacado::Fad::Exp::ExprSpecDefault>’} to ‘double’ in assignment
` 

will be helpful .

Many Thanks
Pushkar 

PS - I did manage to resolve the issue by changing the data type in the definition of the function

**Comment by Saber-tgb** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
